app/models/follow_up.rb

```diff
@@ -70,6 +70,6 @@ def human_status
   end
 
   def current_pending_rdv
-    rdvs.pending.order(starts_at: :asc).first
+    rdvs.select(&:pending?).min_by(&:starts_at)
```

Comment:
j'ai oublié lors de la première review mais il faut itérer sur les `participations` au lieu des `rdvs` ici pour avoir les bons statuts. Il faudra quand même les préloader dans le controller de pour ne pas avoir de N+1